plugin/src/main/java/org/openrewrite/gradle/isolated/DefaultProjectParser.java

```diff
@@ -682,7 +682,7 @@ public List<SourceFile> parse(Project subproject, Set<Path> alreadyParsed, Execu
                     }
                 }
 
-                if(subproject.getPlugins().hasPlugin("org.jetbrains.kotlin.jvm")) {
```

Comment:
I would be very surprised if this leads to a single new kotlin source being parsed. 

If you scroll up, you'll see we're querying the Java plugin's concept of `SourceSet`, which the kotlin JVM plugin builds on top of, to extract information. This is inside a loop over the Java `SourceSet`s. 

I don't believe the kotlin multiplatform, native, or js plugins are built on top of the java plugin at all.
They likely have an analogous concept with a distinct implementation from java source sets.